### PR TITLE
Add a mechanism for using an alternate project template and/or refreshin...

### PR DESCRIPTION
**Pull request by wolfe**
Add a mechanism for using an alternate project template and/or refreshing current project

### CODE REVIEW

File: new_project.sh [hunk 1]
```diff
@@ -3,19 +3,36 @@
...
+    echo "Usage:"
+    echo
+    echo "  new_project.sh"
+    echo "  new_project.sh PROJECT_NAME"
+    echo "  new_project.sh PROJECT_NAME DJANGO_PROJECT_TEMPLATE"
+    echo
+    exit 1
+fi
+
+if [[ -z "$TEMPLATE" ]]; then
```
Comment (bremac) [inline new_project.sh]: Which case does this catch? It seems like the only way for the template location to be empty is if the second argument is both present and empty, in which case setting it to the second argument shouldn't have any impact.

File: setup.sh [hunk 2]
```diff
@@ -10,10 +10,35 @@ function echo_exit {
...
+    # Re-run new_project.sh on specified django_base;
+    # Overwrite local files and exit
+    DIR=`pwd`
+    cd /tmp
+    rm -rf $PROJECT
+    $1/new_project.sh $PROJECT $1
+    cd /tmp/$PROJECT
+    tar cf - . | (cd $DIR; tar xf -)
+    echo "========================"
+    echo "i.e., Re-run ./setup.sh"
```
Comment (bremac) [inline setup.sh:29]: Any reason to not just re-run it in here?

File: new_project.sh [hunk 3]
```diff
@@ -3,19 +3,36 @@
...
 
+TEMPLATE=https://github.com/SheepDogInc/django-base/archive/master.zip
 
 function echo_exit {
     echo ABORTING: $*
     exit 1
 }
 
-if [ -n "$VIRTUALENV" ]; then
+if [ -n "$VIRTUAL_ENV" ]; then
```
Comment (bremac) [inline new_project.sh:13]: Good catch...

File: new_project.sh (hunk 1 above)
Comment (wolfe) [inline new_project.sh]: None -- historical crap.  Removing.

File: setup.sh (hunk 2 above)
Comment (wolfe) [inline setup.sh:29]: I thought it likely I would want to do a "git diff" before re-running the setup.

Comment (bremac) [inline setup.sh:29]: Makes sense

**Comment by bremac** [comment]
:hammer:

**Pull request merged**